content/post/comparing-stash-and-kubestash/index.md

```diff
@@ -0,0 +1,67 @@
+- In Stash, you need to specify the backend information (where the backed up data will be stored) in each Repository. This means repeating the backend information in every Repository, even if it's the same backend. To simplify this, KubeStash introduces the [BackupStorage](https://kubestash.com/docs/v2024.3.16/concepts/crds/backupstorage/) CRD. It holds backend information for a specific backend and can be referenced in multiple Repositories. This allows users to reuse the same backend within a namespace or across the cluster.
+- KubeStash can sync your Repositories between your storage backend and your cluster. Now, when you create a `BackupStorage` object with respective backend information, KubeStash will automatically create the respective Repositories object that were backed up in that storage. This makes restoring into a new cluster much more convenient as you no longer need to create the respective repository manually.
+- In KubeStash, we've introduced a new CRD named [Snapshot](https://kubestash.com/docs/v2024.3.16/concepts/crds/snapshot/) which represents the state of a backup run for one or more components of an application. For every `BackupSession`, it creates `Snapshot` CRs. If a `BackupSession` involves multiple repositories, a `Snapshot` is created for each repository. This makes the task easier to restore specific snapshot and components of an application. Restoring a specific snapshot in Stash was more complex and tiresome process.
+- In KubeStash, we've introduced a new CRD named [RetentionPolicy](https://kubestash.com/docs/v2024.3.16/concepts/crds/retentionpolicy/) that allows you to set how long you’d like to retain the backup data. It offers more control over snapshot cleanup policies compared to Stash. It is also reusable across namespaces. In contrast, in Stash, you need to provide the retention policy separately in every BackupConfiguration.
+- In KubeStash, we've introduced a new CRD named [HookTemplate](https://kubestash.com/docs/v2024.3.16/concepts/crds/hooktemplate/) which specifies a template for an action that will be executed before or/and after backup/restore process. It provides better control over the backup process by letting you use custom variable in the `BackupBlueprint`. It is also reusable across namespaces. In contrast, in Stash, you need to specify the hook separately in every `BackupConfiguration`/`RestoreSession`.
+- KubeStash supports Point-In-Time Recovery (PITR). This feature is not available in Stash.
+
+These are some key features that have been improved or added in KubeStash. Additionally, there are numerous other features that have been enhanced or incorporated into KubeStash. To learn more visit [KubeStash docs](https://kubestash.com/docs/v2024.3.16/welcome/).
```

Comment:
While the above highlights some of the key features and improvements of KubeStash, explore the KubeStash [documentation](https://kubestash.com/docs/v2024.3.16/welcome/) for a comprehensive view of all the new features.